sw/host/rom_ext_image_tools/config.hjson

```diff
@@ -0,0 +1,71 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  fields : [
+    { name: "manifest_identifier",
+      value: "43981",
+      data_type: "int",
+    },
```

Comment:
Can fields be repeated? Why isn't the encoding
```
fields: [
  manifest_identifier: { ... }
]
```

(This shouldn't be terribly hard to do with Serde.)

Is there a reason that there is a top-level `fields` key, rather than having the field list be the top-level document?